shumai/tensor/tensor.ts

```diff
@@ -280,6 +280,11 @@ export class Tensor {
     )
   }
 
+  private track_tidy_ptr(ptr?: number) {
```

Comment:
let's avoid defining this function entirely :)

`_tidyTracker.set(this.ptr, this)` should work in all places where `this.track_tidy_ptr()` is called (with the small perf penalty that we can fix in a followup diff)

Reply:
Yea, if we can replace `get ptr` logic with `return this._ptr`, that'll be far more performant; will remove this now!